backend/src/app.module.ts

```diff
@@ -5,10 +5,12 @@ import { EmploymentContractModule } from './database/employmentContract/employme
 import { AppController } from './app.controller';
 import { LangController } from './lang/lang.controller';
 import { DialogFlowService } from './lang/dialog-flow.service';
+import { EmployeeService } from 'lang/employee.service';
```

Comment:
Aus gegebenem Anlasse (CD ist gestern deswegen nicht gelaufen) - Bitte keine "absoluten" Pfade mehr verwenden, da sonst in der CD Fehler auftreten können. Durch "./lang/employee.service" ersetzten.